Tests/SwerverTests/ActionSpec/HeadActionSpec.swift

```diff
@@ -0,0 +1,22 @@
+import Quick
+import Nimble
+import Swerver
+
+class HeadActionSpec: QuickSpec {
+    
+    override func spec() {
+        describe("A GET action") {
```

Comment:
Copy/paste error? Should this be "A HEAD action"?

Reply:
will fix that